rust/tool/src/cli.rs

```diff
@@ -39,6 +35,10 @@ struct InstallCommand {
     #[arg(long, default_value_t = 1)]
     configuration_limit: usize,
 
+    /// EFI Bootloader Path (e.g. systemd/lib/systemd/boot/efi/systemd-bootx64.efi)
+    #[arg(long)]
+    efi_boot_path: PathBuf,
```

Comment:
lzbt would need two different modes, one for systemd-boot and one for grub to generate the right configs.

Reply:
Do we want two modes specific for those two boot loaders? If we add support to other things like ext/syslinux later on, wouldn't it make more sense to have a standard interface and then specifics for each bootloaders? E.g. a config generator that could be extended for systemd and grub, an EFI path that would work for any given EFI binary, etc.